src/common/helper.js

```diff
@@ -1082,9 +1090,10 @@ async function getUserById (userId, enrich) {
   const user = _.pick(res.body, ['id', 'handle', 'firstName', 'lastName'])
 
   if (enrich) {
-    user.skills = (res.body.skills || []).map((skillObj) =>
-      _.pick(skillObj.skill, ['id', 'name'])
-    )
+    user.skills = await Promise.all((res.body.skills || []).map(async (userSkill) => {
+      const skill = await getSkillById(userSkill.skillId)
```

Comment:
Same as with above comment for `listUsersByExternalId`. We should not make separate call for each skill.